web/core/components/issues/workspace-draft/draft-issue-block.tsx

```diff
@@ -0,0 +1,127 @@
+            "md:flex-row md:items-center": isSidebarCollapsed,
+            "lg:flex-row lg:items-center": !isSidebarCollapsed,
+          }
+        )}
+      >
+        <div className="flex w-full truncate">
+          <div className="flex flex-grow items-center gap-0.5 truncate">
+            <div className="flex items-center gap-1">
+              {/* {displayProperties && (displayProperties.key || displayProperties.issue_type) && ( */}
+              <div className="flex-shrink-0">
+                {issue.project_id && (
+                  <div className="flex items-center space-x-2">
+                    <IdentifierText
+                      identifier={projectIdentifier}
+                      enableClickToCopyIdentifier
+                      textContainerClassName="text-xs font-medium text-custom-text-300"
+                    />
+                  </div>
+                )}
+              </div>
+              {/* )} */}
+
+              {/* sub-issues chevron */}
+              <div className="size-4 grid place-items-center flex-shrink-0" />
+            </div>
+
+            <Tooltip
+              tooltipContent={issue.name}
+              // isMobile={isMobile}
+              position="top-left"
+              // disabled={isCurrentBlockDragging}
+              renderByDefault={false}
+            >
+              <p className="w-full truncate cursor-pointer text-sm text-custom-text-100">{issue.name}</p>
+            </Tooltip>
+          </div>
+
+          {/* quick actions */}
+          <div
+            className={cn("block border border-custom-border-300 rounded", {
+              "md:hidden": isSidebarCollapsed,
+              "lg:hidden": !isSidebarCollapsed,
+            })}
+          >
+            <WorkspaceDraftIssueQuickActions
+              parentRef={issueRef}
+              issue={issue}
+              handleUpdate={async (data) => updateIssue(workspaceSlug, issueId, data)}
+              handleDelete={async () => deleteIssue(workspaceSlug, issueId)}
+              handleMoveToIssues={async () => moveIssue(workspaceSlug, issueId, issue)}
+            />
+          </div>
+        </div>
+
+        <div className="flex flex-shrink-0 items-center gap-2">
+          <DraftIssueProperties
+            className={`relative flex flex-wrap ${isSidebarCollapsed ? "md:flex-grow md:flex-shrink-0" : "lg:flex-grow lg:flex-shrink-0"} items-center gap-2 whitespace-nowrap`}
+            issue={issue}
+            updateIssue={async (projectId, issueId, data) => {
+              await updateIssue(workspaceSlug, issueId, data);
+            }}
+            activeLayout="List"
+          />
+          <div
+            className={cn("hidden", {
+              "md:flex": isSidebarCollapsed,
+              "lg:flex": !isSidebarCollapsed,
+            })}
+            onClick={(e) => {
+              e.preventDefault();
+              e.stopPropagation();
+            }}
+          >
+            <WorkspaceDraftIssueQuickActions
```

Comment:
Why is there two `WorkspaceDraftIssueQuickActions`

Reply:
@rahulramesha We have that because we have to render in two different places based on the responsiveness

Reply:
ok